spec/models/feedback_message_spec.rb

```diff
@@ -14,6 +14,8 @@
     it { is_expected.to validate_presence_of(:feedback_type) }
     it { is_expected.to validate_presence_of(:reported_url) }
     it { is_expected.to validate_presence_of(:message) }
+    it { is_expected.to validate_length_of(:reported_url).is_at_most(50) }
```

Comment:
Hey Andy, does it make sense to also include a check for a `:reported_url` that is *more than* 50 characters? Seeing as that was the use case that necessitated this PR?

Reply:
Good question, based on the gem's documentation on this method I think it should cover it:

```
# ##### is_at_most
#
# Use `is_at_most` to test usage of the `:maximum` option. This asserts
# that the attribute can take a string which is equal to or shorter than
# the given length and cannot take a string which is longer.
```

https://github.com/thoughtbot/shoulda-matchers/blob/390c2ca161b35bce7171c65a6ec3b653b7c2b877/lib/shoulda/matchers/active_model/validate_length_of_matcher.rb#L60-L64

Reply:
Great! 👍